docs/user_guide/labels_and_annotations_in_yunikorn.md

```diff
@@ -49,5 +49,7 @@ All annotations are under the namespace `yunikorn.apache.org`. For example `yuni
 | `namespace.quota`                   | Set the maximum capacity of the queue mapped to this namespace. Further details can be found in the [ Resource Quota Management](resource_quota_management#namespace-quota) documentation.                                                                                                      |
 | [DEPRECATED] `namespace.max.cpu`    | Replaced with ``namespace.quota`` since version 1.2.0                                                                                                                                                                                                                                           |
 | [DEPRECATED] `namespace.max.memory` | Replaced with `namespace.quota` since version 1.2.0                                                                                                                                                                                                                                             |
+| `namespace.enableYuniKorn`          | Controls which namespace will have pods forwarded to Yunikorn for scheduler. Further details can be found in the [Service Configuration #admission-controller-filtering-settings](./service_config.md#admission-controller-filtering-settings) documentation.                                                                                                                                                                                                                                           |
+| `namespace.generateAppId`           | Controls which namespaces will have pods labeled with an `applicationId`. Further details can be found in the [Service Configuration #admission-controller-filtering-settings](./service_config.md#admission-controller-filtering-settings) documentation.                                                                                                                                                                                                                                             |
```

Comment:
For both of these new lines, the trailing '|' should be vertically aligned with the rest of the table borders when displayed with a monospace font.